libs/coin-modules/coin-mina/src/signer/getAddress.ts

```diff
@@ -0,0 +1,26 @@
+import { GetAddressFn } from "@ledgerhq/coin-framework/bridge/getAddressWrapper";
+import { SignerContext } from "@ledgerhq/coin-framework/signer";
+import { GetAddressOptions } from "@ledgerhq/coin-framework/derivation";
+import { GetAddressResponse, MinaSigner } from "../types/signer";
+import { getAccountNumFromPath } from "../common-logic";
+import invariant from "invariant";
+import { log } from "@ledgerhq/logs";
+
+const resolver = (signerContext: SignerContext<MinaSigner>): GetAddressFn => {
+  return async (deviceId: string, { path, verify }: GetAddressOptions) => {
+    const account = getAccountNumFromPath(path);
+    log("debug", "getAddress, extracted account for path: ", { account, path, verify });
```

Comment:
Do we really need this log ? There is no specific issue there 🤔

Reply:
Removed, it was remnant from past debugging 😓